packages/video_player/video_player_windows/example/.gitignore

```diff
@@ -0,0 +1 @@
+.vs/
```

Comment:
Is this right? Should we mirror other example `.gitignore`s? For example: https://github.com/flutter/packages/blob/main/packages/url_launcher/url_launcher_windows/example/.gitignore

Reply:
Generally we use the `flutter create`d `.gitignore` without changes for the example apps.

Reply:
I believe this should be added, maybe to the root .gitignore. Since Flutter generates Visual Studio projects for Windows, whenever I open the project files, it automatically creates the .vs folder, and it tries to check it in. At least in the samples it does.
It is added on the recommended .gitignore for Visual Studio: https://github.com/github/gitignore/blob/main/VisualStudio.gitignore#L36C4-L36C5

Reply:
How are you opening them? The generated projects are in the `build` folder, which is already excluded.

Reply:
I see. It wasn't doing that since I originally used the .gitignore from the Android project here:
https://github.com/flutter/packages/blob/main/packages/video_player/video_player_android/example/.gitignore
I can use the default one.